tests/integration/test_k8s.py

```diff
@@ -136,21 +136,32 @@ async def test_remove_worker(kubernetes_cluster: model.Model):
 
 
 @pytest.mark.abort_on_fail
-async def test_remove_non_leader_control_plane(kubernetes_cluster: model.Model):
+async def test_remove_non_leader_control_planes(kubernetes_cluster: model.Model):
     """Deploy the charm and wait for active/idle status."""
     k8s = kubernetes_cluster.applications["k8s"]
     worker = kubernetes_cluster.applications["k8s-worker"]
     expected_nodes = len(k8s.units) + len(worker.units)
     leader_idx = await get_leader(k8s)
     leader = k8s.units[leader_idx]
     follower = k8s.units[(leader_idx + 1) % len(k8s.units)]
+    follower2 = k8s.units[(leader_idx + 2) % len(k8s.units)]
```

Comment:
let's just gather a list of the followers

```suggestion
    followers = [unit for idx, unit in enumerate(k8s.units) if idx !=leader_idx ]
```